types/ember-data__adapter/index.d.ts

```diff
@@ -1,11 +1,11 @@
-// Type definitions for @ember-data/adapter 3.15
+// Type definitions for @ember-data/adapter 3.12
```

Comment:
If we walk the version _back_ for the `@ember-data/*` packages, even if the `latest` tag points to 3.12.x, isn't there a danger of folks winding up with an older `3.15.x` version in a some cases?

Reply:
Good point. Accordingly, and given 3.12 is in security-fixes-only LTS, I went ahead and bumped to 3.16 for all the core Ember and Ember Data types!